packages/core/src/components/Table/Table/Table.types.ts

```diff
@@ -1 +1 @@
-export type RowSizes = "medium" | "large";
+export type RowSizes = "medium" | "large" | "small";
```

Comment:
```suggestion
export type RowSizes = "small" | "medium" | "large";
```

I'm sorry for being petty 😆

Reply:
lol no worries, good point